unix/fixed.c

```diff
@@ -0,0 +1,87 @@
+// 9 june 2016
+#include "uipriv_unix.h"
+
+struct fixedChild {
+	uiControl *c;
+};
+
+struct uiFixed {
+	uiUnixControl c;
+	GtkWidget *widget;
+	GtkContainer *container;
+	GtkFixed *fixed;
```

Comment:
Don't use GtkFixed; it does not properly theme with CSS. GtkLayout would be better, but the ideal solution would be to have a custom GtkContainer subclass that does the barest minimum possible. Look at the version of package ui that immediately predated libui for details. (I can do this later too.)

Reply:
I'll look at that.

Reply:
Ok I finally got more time to look at this. I assume you're talking about this: https://github.com/andlabs/ui/blob/de9d72299fb89a8b6cdc8963cd6b6ae708a81e80/container_unix.c
So you're overriding GtkContainer, but where do you actually set children size and location?

Reply:
Line 62, which is a Go function. I forget what you call in there for each child control, but you can check the Go code.